### PR TITLE
Khwang/readme

### PR DESCRIPTION
**Pull request by khwang**
Formats the README to be nicer

---
This change is [Reviewable](https://reviewable.io/reviews/khwang/honeydew/8)

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -1,2 +1,17 @@
-{<img src="https://travis-ci.org/khwang/honeydew.svg?branch=master" alt="Build Status" />}[https://travis-ci.org/khwang/honeydew]
+= Honeydew
+
+Home :: https://github.com/khwang/honeydew
+Bugs/Feature Requests :: https://github.com/khwang/honeydew/issues
+Build Status :: {<img src="https://travis-ci.org/khwang/honeydew.svg?branch=master" alt="Build Status" />}[https://travis-ci.org/khwang/honeydew]
+
+== About
+
+Honeydew is a site designed to help people achieve financial fitness.
```
Comment (Radahika) [inline README.rdoc]: This needs elaboration. What does financial fitness mean? Maybe take some ideas from mint and r/financialindependence?

File: README.rdoc [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+
+== Getting Started
+
+1. Clone the repo: <code>git clone https://github.com/khwang/honeydew</code>
```
Comment (Radahika) [inline README.rdoc]: What ruby version do you need to run honeydew? Should we specify that?

File: README.rdoc [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+2. Install dependencies: <code>bundle install</code>
+3. Start the server: <code>bin/rails server</code>
```
Comment (Radahika) [inline README.rdoc]: Don't you need to do something with the database?
I suggest you actually follow these steps in a clean directory and figure out what needs to be done.

File: README.rdoc (hunk 2 above)
Comment (khwang) [inline README.rdoc]: No, this is specified in our .ruby-version file

File: README.rdoc (hunk 3 above)
Comment (khwang) [inline README.rdoc]: Done

File: README.rdoc (hunk 1 above)
Comment (khwang) [inline README.rdoc]: I figure people will just read the about page for more info/this isn't fully fleshed out. I mostly wanted us to get the nice badges :)

**Pull request merged**